src/views/ContributorRewards/components/MonthlyClaim.tsx

```diff
@@ -16,12 +16,16 @@ const MonthlyClaim: React.FC = () => {
     return <Button disabled full text='Claim' variant='secondary' />
   }, [status, onClaimRewards, claimableQuantity])
 
+  const onSelectMonth = (month: string) => {
+    setMonth(month)
+  }
+
   return (
     <>
       <Card>
         <CardContent>
           <Spacer size='sm' />
-          <MonthsDropdown />
+          <MonthsDropdown onSelectMonth={onSelectMonth} />
```

Comment:
Looks like `onSelectMonth` is unused since `MonthsDropdown` imports its own `useRewards`. Probably best to keep this here, add in month as well as a prop and remove `useRewards` from the `MonthsDropdown`

Reply:
Would it not be easier to just remove all references to `onSelectMonth`? I had originally been using the onSelectMonth strategy to lift the state of MonthsDropdown and it was much more code, which is why I changed it.

Reply:
Sure, you could remove all references to `onSelectMonth` too. Just do one or the other cause atm it's not being used in `MonthsDropdown`. The case for keeping it as props is incase you want to reuse `MonthsDropdown` somewhere else but that case doesn't seem super likely